drivers/sonar_i2cxl.hpp

```diff
@@ -43,7 +43,7 @@
 #ifndef I2CXL_SONAR_H_
```

Comment:
done in eb5fddfc6a91cceafdb50b47dba4ff9a82dc44a4done in eb5fddfc6a91cceafdb50b47dba4ff9a82dc44a4